zookeeper-contrib/zookeeper-contrib-zooinspector/pom.xml

```diff
@@ -96,4 +96,15 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources</directory>
+        <includes>
```

Comment:
is this really needded ? isn't it the default ?

Reply:
if remove it , mvn compile will faile for 
`'build.resources.resource.directory' is missing.`

Reply:
after doing some tests, got conclution as follow
if not set resource, will use parent resource config
if add one resource element config, parent config will not show in "effective pom"
so actually, the include is not necessary, but direcotry is required
will do a new commit remove the include